### PR TITLE
rsocket-py documentation - added rxpy examples

### PR DESCRIPTION
**Pull request by gabis-precog**
rsocket-py documentation - added rxpy examples

### Motivation:

To add rxpy examples

### Modifications:

Added rxpy examples

### Result:

documentation has rxpy examples

### CODE REVIEW

File: content-docs/guides/rsocket-py/rxpy/introduction.mdx [hunk 1]
```diff
@@ -9,3 +9,91 @@ sidebar_label: Introduction
...
+        ... # Execute requests
+
+if __name__ == '__main__':
+    asyncio.run(main())
+```
+
+## Examples
+
+RxRSocket can be used as a context manager with a client which is not yet connected. It will close the underlying client
+when existing the context
```
Comment (viglucci) [inline content-docs/guides/rsocket-py/rxpy/introduction.mdx:37]: Small typo: `existing` -> `exiting`

File: content-docs/guides/rsocket-py/rxpy/introduction.mdx [hunk 2]
```diff
@@ -9,3 +9,91 @@ sidebar_label: Introduction
...
+
+sent_payloads = [Payload(data) for data in [b'1', b'2', b'3']]
+
+received_messages = await rx_client.request_channel(
+        Payload(b'request text'),
+        observable=rx.from_list(sent_payloads),
+        request_limit=2).pipe(
+    operators.map(lambda payload: payload.data),
+    operators.to_list()
+)
```
Comment (viglucci) [inline content-docs/guides/rsocket-py/rxpy/introduction.mdx:98]: Small formatting adjustment:

```suggestion
from rsocket.payload import Payload
import rx
from rx import operators

sent_payloads = [Payload(data) for data in [b'1', b'2', b'3']]

received_messages = await rx_client.request_channel(
    Payload(b'request text'),
    observable=rx.from_list(sent_payloads),
    request_limit=2
).pipe(
    operators.map(lambda payload: payload.data),
    operators.to_list()
)
```

**Comment by viglucci** [comment]
I'm not overly familiar with python so I can't speak to the validness of the examples, but the addition of the page LGTM.

File: content-docs/guides/rsocket-py/rxpy/introduction.mdx (hunk 1 above)
Comment (gabis-precog) [reply content-docs/guides/rsocket-py/rxpy/introduction.mdx]: done

File: content-docs/guides/rsocket-py/rxpy/introduction.mdx (hunk 2 above)
Comment (gabis-precog) [reply content-docs/guides/rsocket-py/rxpy/introduction.mdx]: fixed.

**Pull request merged**